scripts/manage-db.py

```diff
@@ -86,7 +87,26 @@ def cleanup_releases_history(trans, dryrun=True):
     print "Total Deleted: %d" % total_deleted
 
 
-def extract_active_data(url, dump_location='dump.sql'):
+def _extract_partials(json_object):
+    """Returns a generator that contains \
+    all value objects having key as 'partials'"""
+    if isinstance(json_object, dict):
+        for key, value in json_object.iteritems():
+            if key == "partials":
```

Comment:
Don't forget about old-style blobs, which have a singular "partial" block instead of a list of them. Eg: https://github.com/mozilla/balrog/blob/5d75609ed75b7a349b0ca5ba5faf6cd8940af775/aus-data-snapshots/3.6.12-3.6.13/Firefox-3.6.13-build3.json